service/src/main/kotlin/fi/tampere/trevaka/invoice/config/InvoiceConfiguration.kt

```diff
@@ -145,7 +145,7 @@ class TampereInvoiceProductProvider : InvoiceProductProvider {
             Product.DAYCARE to FeeAlteration.Type.RELIEF,
             Product.PRESCHOOL_WITH_DAYCARE to FeeAlteration.Type.DISCOUNT,
             Product.PRESCHOOL_WITH_DAYCARE to FeeAlteration.Type.RELIEF ->
-                Product.DAYCARE_DISCOUNT
```

Comment:
Should

```
Product.DAYCARE to FeeAlteration.Type.DISCOUNT,
Product.DAYCARE to FeeAlteration.Type.RELIEF,
```

still mapped to `Product.DAYCARE_DISCOUNT`? Like:

```
Product.DAYCARE to FeeAlteration.Type.DISCOUNT,
Product.DAYCARE to FeeAlteration.Type.RELIEF ->
    Product.DAYCARE_DISCOUNT
Product.PRESCHOOL_WITH_DAYCARE to FeeAlteration.Type.DISCOUNT,
Product.PRESCHOOL_WITH_DAYCARE to FeeAlteration.Type.RELIEF ->
    Product.PRESCHOOL_WITH_DAYCARE_DISCOUNT
```

Reply:
Yep, 🤦🏻‍♂️